t/22-dashboard.t

```diff
@@ -100,8 +101,15 @@ like(
 );
 
 $t->get_ok('/dashboard_build_results')->status_is(200);
-@h2 = $t->tx->res->dom->find('h2 a')->map('text')->each;
-is_deeply(\@h2, ['opensuse test', 'Test parent'], 'parent group shown and opensuse is no more on top-level');
+@h2 = $t->tx->res->dom->find('h2 a')->map(sub ($e) { $e->text . ($e->attr('title') // '') })->each;
+is_deeply(\@h2, ['opensuse test', 'Test parent', 'Shows the latest test results for all job groups within this parent job group'], 'parent group shown and opensuse is no more on top-level');
+$t->element_exists('#test_result_overview_link_1');
```

Comment:
This test has no description but it is useless anyways because if that one fails the next test would fail as well. (It would make more sense to check whether `$tests_overview_dashboard` before blindly calling a method of it.)

Reply:
Do you think this new version is better?

Reply:
New version looks better